smalltalksrc/VMMaker/SlangAbstractTestCase.extension.st

```diff
@@ -4,6 +4,7 @@ Extension { #name : #SlangAbstractTestCase }
 SlangAbstractTestCase >> setUp [
 	super setUp.
 	ccg := MLVMCCodeGenerator new.
+	ccg vmClass: VMClass. 
```

Comment:
Had to add this since this methods needs a reference to a VMClass

```smalltalk
shouldGenerateStruct: structClass

^ (super shouldGenerateStruct: structClass)
		and: [self vmClass shouldGenerateTypedefFor: structClass]
```